overrides/scripts/roadstuff.zs

```diff
@@ -37,16 +32,4 @@ furnace.addRecipe(<chisel:concrete>, <roadstuff:concreteBlock>);
 
 
 // Aluminium Rod
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <minecraft:shears>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsWood>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsStone>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsDiamond>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsGold>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsCopper>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsTin>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsSilver>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsLead>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsNickel>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsInvar>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsBronze>.anyDamage().transformDamage()]);
-recipes.addShapeless(<roadstuff:itemIronPart>*2, [<ore:ingotAluminium>, <ThermalFoundation:tool.shearsPlatinum>.anyDamage().transformDamage()]);
\ No newline at end of file
+recipes.addShaped(<roadstuff:itemIronPart>*2, [[<ore:ingotAluminium>, null], [<ore:ingotAluminium>, null]]);
```

Comment:
Does this also work on the right side of crafting tables?

Reply:
fixed